sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/json/JsonNodeType.java

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.cosmos.implementation.json;
+
+/**
+ * The enumeration of JSON node types.
+ */
+enum JsonNodeType {
+    /**
+     * Corresponds to the 'null' value in JSON.
+     */
+    NULL,
```

Comment:
>NULL [](http://example.com/codeflow?start=4&length=4)

Is all caps the naming convention here?

Reply:
It's the Java convention and seems to be followed in some (but not all) of the SDK enum types.

Reply:
@kushagraThapar any preference here?